app/globals.scss

```diff
@@ -23,11 +23,7 @@ body {
 }
 
 .container {
-  max-width: 1180px;
-  min-height: 100vh;
+  // max-width: 1128px;
```

Comment:
**nitpick:** Container should always have maxWidth (which is specified in Figma) + it should be centered with `margin: 0 auto`